weaviate/util.py

```diff
@@ -709,6 +709,68 @@ def parse_version_string(ver_str: str) -> tuple:
         )
 
 
+class _ServerVersion:
+    major: int
+    minor: int
+    patch: int
+
+    def __init__(self, major: int, minor: int, patch: int) -> None:
+        self.major = major
+        self.minor = minor
+        self.patch = patch
```

Comment:
class.Properties

Reply:
As in, the types of the attrs? Is this not inferred by the function args?